packages/js/components/src/table/empty.tsx

```diff
@@ -1,39 +1,32 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import { createElement } from '@wordpress/element';
+import React from 'react';
+
+type EmptyTableProps = {
+	children: React.ReactNode;
+
+	// An integer with the number of rows the box should occupy.
```

Comment:
Suggest changing it to JSDoc style

```suggestion
	/** An integer with the number of rows the box should occupy. */
```